### DISCUSSION TITLE
energy loss function coeffs for dataset with variable system size

### DISCUSSION THREAD

**Discussion by saeedshahi** [mir-group/nequip#228] [Q&A] [answered]
Hi,

I have a dataset that is combined different system sizes like 40 atoms for the alpha phase and 160 atoms for another phase.
What is the best choice for energy loss function coeffs? and what happens if I use default PerAtomMSELoss? is it the right choice and what is the meaning of training_e/N_mae for variable system size?

Best,
Saeed

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @saeedshahi ,

Exactly, `PerAtomMSELoss` is the correct choice for training on energy when the system size is variable. The energy error for each frame is simply normalized by dividing by the number of atoms in the frame before averaging over frames in the batch.
